components/eam/src/physics/crm/crmclouds_camaerosols.F90

```diff
@@ -1,135 +1,110 @@
   use ndrop,            only: dropmixnuc
-  use modal_aero_data
+  use phys_control,     only: phys_getopts
+  use modal_aero_data,  only: ntot_amode, nspec_amode, numptr_amode, lmassptr_amode
   use rad_constituents, only: rad_cnst_get_info
-
-  !!! Input 
-  type(physics_state), intent(in)    :: state               ! state variables
-  type(physics_buffer_desc), pointer :: pbuf(:)             ! physics buffer
-  real(r8), intent(in) :: pblht(pcols)                      ! PBL height (meter)
-  real(r8), intent(in)  :: dtime                            ! timestep
-  real(r8), intent(in) :: cflx(pcols,pcnst)                 ! constituent flux from surface
-  real(r8), intent(in) :: wwqui_cen(pcols, pver)            ! vertical velocity variance in quiescent class (m2/s2)
-  real(r8), intent(in) :: wwqui_cloudy_cen(pcols, pver)     ! vertical velocity variance in quiescent, and cloudy class (m2/s2)
-  real(r8), intent(in) :: wwqui_bnd(pcols, pver+1)          ! vertical velocity variance in quiescent class (m2/s2)
-  real(r8), intent(in) :: wwqui_cloudy_bnd(pcols, pver+1)   ! vertical velocity variance in quiescent, and cloudy class (m2/s2)
-  integer,  intent(in) :: species_class(:)
-
-  !!! output
-  type(physics_ptend), intent(out) :: ptend   ! package tendencies
-
-  !!! Local variables
-  integer i,k,m, k1, k2
-  integer ifld, itim
-  integer ixcldliq, ixcldice, ixnumliq
-  integer l,lnum,lnumcw,lmass,lmasscw
-
-  integer :: lchnk    ! chunk identifier
-  integer :: ncol     ! number of atmospheric columns
+  !-----------------------------------------------------------------------------
+  ! Input 
+  type(physics_state), intent(in)               :: state             ! state variables
+  type(physics_buffer_desc), pointer            :: pbuf(:)           ! physics buffer
+  real(r8), intent(in), dimension(pcols)        :: pblht             ! PBL height (m)
+  real(r8), intent(in)                          :: dtime             ! timestep
+  real(r8), intent(in), dimension(pcols,pcnst)  :: cflx              ! constituent flux from surface
+  real(r8), intent(in), dimension(pcols,pver)   :: wwqui_cen         ! vert velocity variance in quiescent class        (m2/s2)
+  real(r8), intent(in), dimension(pcols,pver)   :: wwqui_cloudy_cen  ! vert velocity variance in quiescent+cloudy class (m2/s2)
+  real(r8), intent(in), dimension(pcols,pver+1) :: wwqui_bnd         ! vert velocity variance in quiescent class        (m2/s2)
+  real(r8), intent(in), dimension(pcols,pver+1) :: wwqui_cloudy_bnd  ! vert velocity variance in quiescent+cloudy class (m2/s2)
+  integer,  intent(in)                          :: species_class(:)  ! 
+  ! Output
+  type(physics_ptend), intent(out)              :: ptend             ! output tendencies
+  !-----------------------------------------------------------------------------
+  ! Local variables
+  integer :: i,k,m, k1, k2
+  integer :: itim
+  integer :: ixcldliq, ixcldice, ixnumliq
+  integer :: l,lnum,lnumcw,lmass,lmasscw
```

Comment:
So long as you're changing all the formatting, you may as well be consistent about spaces after commas.

Reply:
good call, fixed.